backend/src/core/models/user/user.ts

```diff
@@ -39,6 +41,13 @@ export class User extends Model<User> {
   @Column({ type: DataType.STRING, allowNull: true })
   apiKeyHash: string | null
 
+  @Column({
```

Comment:
I thought making it non-nullable and setting a default value (rather than relying on env vars) is a better way to manage our rate limit